xLights/effects/FacesEffect.cpp

```diff
@@ -1084,8 +1131,17 @@ void FacesEffect::RenderFaces(RenderBuffer& buffer,
     } else if (phoneme == "rest" || phoneme == "(off)") {
             if ("Auto" == eyes) {
                 if ((buffer.curPeriod * buffer.frameTimeInMs) >= cache->nextBlinkTime) {
-                    // roughly every 5 seconds we'll blink
-                    cache->nextBlinkTime += intRand(4500, 5500);
+                    //calculate the blink time, taking into account user selection
```

Comment:
So this code is repeated 5 times in this file,  would be better as a function.

Reply:
I can do this - I just copied the existing code pattern but can optimize as you say

Reply:
addressed comment